templates/coderd.yaml

```diff
@@ -232,21 +206,14 @@ data:
   privateKey: {{ genPrivateKey "ecdsa" | b64enc }}
   {{- end }}
 {{- end }}
-{{ if eq (include "coder.useServiceNext" .) "true" }}
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  {{- if (lookup "v1" "Service" .Release.Namespace "ingress-nginx") }}
-  name: ingress-nginx
-  {{- else }}
   name: {{ include "coder.serviceName" . }}
-  {{- end }}
```

Comment:
I don't think we can ever get rid of this without causing load balancer IPs to change for all customers

Reply:
Will add it back. I think we should recommend that people add a reserved static IP and add the appropriate annotation though, to be safe